docs/x509.rst

```diff
@@ -181,6 +181,71 @@ X.509 Certificate Object
 
         For version 3 X.509 certificates.
 
+.. class:: Attribute
+
+    .. versionadded:: 0.8
+
+    .. attribute:: oid
+
+        :type: :class:`ObjectIdentifier`
+
+        The attribute OID.
+
+    .. attribute:: value
+
+        :type: :class:`str`
```

Comment:
I made them text since they're a dotted representation, so they're a `str` on py2 and py3.